### DISCUSSION TITLE
Bitrix24 Forms with Nuxt 3

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#18045] [Questions] [answered]
I am trying to insert a form from bitrix into my Vue component.
`./components/Btrx24.vue`
```javascript
<script lang="jsx">
	export default {
		render() {
			return h('div', {}, [
				h(
					'script',
					{ 'data-b24-form': 'click/12/7uz2pa', 'data-skip-moving': true },
					`
					(function (w, d, u) {
						var s = d.createElement('script');
						s.async = true;
						s.src = u + '?' + ((Date.now() / 180000) | 0);
						var h = d.getElementsByTagName('script')[0];
						h.parentNode.insertBefore(s, h);
					})(window, document, 'https://bitrix.santrek.ru/upload/crm/form/loader_12_7uz2pa.js');
				`,
				),
				h('a', {}, 'Some Link'),
			]);
		},
	};
</script>
```
`tsconfig.json`
```javascript
{
	"extends": "./.nuxt/tsconfig.json",
	"compilerOptions": {
		"noImplicitAny": false,
		"jsx": "preserve"
	}
}
```
---
In the description of Bitrix it is said to insert their automatically generated <script> tag before the link or button
```javascript
<script data-b24-form="click/12/7uz2pa" data-skip-moving="true">
  (function (w, d, u) {
    var s = d.createElement('script');
    s.async = true;
    s.src = u + '?' + ((Date.now() / 180000) | 0);
    var h = d.getElementsByTagName('script')[0];
    h.parentNode.insertBefore(s, h);
  })(
    window,
    document,
    'https://bitrix.santrek.ru/upload/crm/form/loader_12_7uz2pa.js',
  );
</script>
```
---

# How is it supposed to work:
# [Run Here](https://codesandbox.io/s/empty-browser-vt3sf)

---

# What happens in the end is that the tags are created correctly, but the bitrix form is not created and, accordingly, is not displayed.
![image](https://user-images.githubusercontent.com/25690524/148912709-9db86e7a-2787-4b4f-a45d-5816277279e9.png)
# [Run Here](https://codesandbox.io/s/nostalgic-wescoff-vcw7d?file=/pages/index.vue)

---
# Please Help. Thx

**Comment by mihail727** [comment, marked as answer]
this works only in production mode....

**Comment by aspisklov** [comment]
Спасибо, тебе за тему! Весь день убил, пока твою тему не нашел:)))
